### DISCUSSION TITLE
How to change the background color of the folding column in a terminal theme?

### DISCUSSION THREAD

**Discussion by davidvontamar** [orbitalquark/textadept#269] [Q&A] [answered]
I'm trying to create a bright terminal theme with white background. I can't find the property that changes the background of the folding column. Here's what I've tried so far, with partial success:

```
for i = 1, 256 do
	view.marker_fore[i] = colors.black
	view.marker_back[i] = colors.light_white
	view.marker_back_selected[i] = colors.white
end
```

Here's the result:
![image](https://user-images.githubusercontent.com/24723374/192980703-a1c20e6e-ac21-4f4c-9fbc-d757094c45ae.png)

**Comment by gabdub** [comment]
try:
`view:set_fold_margin_color(true, colors.light_white)`

**Reply by davidvontamar** [reply]
I'm sorry, but this doesn't seem to do anything. Does it work for you?

**Reply by gabdub** [reply]
It works in the graphic version. It's actually two colors intertwined.

![image](https://user-images.githubusercontent.com/11639609/193143347-bbf73bc2-9a54-43b6-ba53-c5991b77c897.png)

**Comment by orbitalquark** [comment]
It's quite possible that this has not been implemented in the terminal version. I can't get it to work either.

**Reply by davidvontamar** [reply]
> It's quite possible that this has not been implemented in the terminal version. I can't get it to work either.

I want to use the terminal version but I can't do without a bright theme. Could it be implemented eventually?

**Comment by orbitalquark** [comment]
I don't know, maybe. I just inverted my terminal's black and white color settings to have a "light" theme using Textadept's default terminal theme.

**Comment by orbitalquark** [comment]
It's quite possible that this has not been implemented in the terminal version. I can't get it to work either.

**Comment by orbitalquark** [comment, marked as answer]
I've committed a change to Scinterm that supports `view:set_fold_margin_color(true, colors.white)` and `view:set_fold_margin_hi_color(true, colors.white)`: https://github.com/orbitalquark/scinterm/commit/9632ce32ab0c7c19828600e746036bfa39b7e061

This should be in the next nightly build of Textadept.

**Reply by davidvontamar** [reply]
Thanks a lot! I really appreciate that.
